lms/djangoapps/support/static/support/jsx/single_support_form.jsx

```diff
@@ -119,6 +121,16 @@ class RenderForm extends React.Component {
   }
 
   render() {
+    if (this.state.success) {
+      return (
+        <Success
+          homepageUrl={this.props.context.homepageUrl}
+          dashboardUrl={this.props.context.dashboardUrl}
+          isLoggedIn={this.props.context.user && true}
```

Comment:
`&& true` does seem needed? Are you trying to cast it into a boolean?

Reply:
default value of `isLoggedIn` is `false`, so I am passing `true` if user is logged in.

Reply:
If the user is set, then the value will be true? Why not test for that case explicitly?

Reply:
no user is a object not a Boolean. If I remove `true` I will get error.

Reply:
Why not check to see if user is undefined then? The intent is clearer then.

Reply:
I don't want to pass `this.props.context.user` to Success component. Sorry I am new on this so not able to understand your suggestion can you write some code what you are suggesting.